ui/StatusQ/src/assets/img/icons/status-connector.svg

```diff
@@ -0,0 +1,27 @@
PehxXwiH1Lz+m/piPLHqS6xODJBOzCK2Bla8nReVoMbDwB022YxGv7eCQPP7KIcwt88f2ke+D+4nIHgLSH9sV51L80kdavZ8HgU0ZxP4xB5ZGlpCMQJBohEO8fRvOWJSQUPTWKi2SddxivZB4sHM//5clIsIhd7T3ksnlA80mlWsBRTlyS9eOYzxl6Yx1BBTEpzvEoRzZdsKqyXXCAkeaZGsfbHbYMoc0g3BpTxzfJZIv/nUevDRe6OrR4hyxiUY77eKRosHu6g4ZyszB3q/r4PAlHL6AAhZAGA3jkSj4VIp4jlDmENGgaArhDqtjVoY6xY5ucj0FHhC5XH8sAcX8OjiEh3eXn67kDaR/vO4zq3CICCBYyK0RJ9/KdcqIVDmOIFIHXz4A4mRcpEji2NLYw7G4br5n8X3Uk5ARkTuWRGbXD6pwFBXrLuUGFGIFFtSAULeBSEfukGw4GBjn4evEPXJkOFAi4SJCb2krxAkrLEIDAxrswGEPPRUe6Kpw0A2Grh/z2/lUCaLsgdw0qBNZz3TF+hHl9gBH1LvhETEJm5RFaIA4FGIdGDRFYw8vxBZT66tXnpDOP2zxsfb1R+fhv7uq+8eciFukmKCC1AxCaWBFgHi/BhFKNxGaChhsSYIlEegUOHJeiQdPjD6wEonNrafFgyvih39O47baZXCisIN/CorIhENYM4McGhA+ItWPLEEcBwsUyePI1gmNPZA84gKIueRVb4B4/WQGAUXZxKLki7NXnpg+u7j1dNVw0FKjMbmAWLPEwXcJGBOLlEniwdePWMGE0sDKtxMYwMKXSNTckkNkESAQHJTLiTyN49UY7bIH1XF0GiyhaPZgsWvTwFKidIuwSmaOg0PKHAIHJ0Kqq6w9QCjcYb3UNqZWGgQWyFj8ZLsc8oTifBHtH8y/Omer9JCMRa3p+xVqMOSQPZd1cPDiC4+KED/5qvCHK49YHQk5IotAxFZGuveNWeSJxYd0BpGlPWipn66SwvTES4vGtW1aaHw/ObAlYQ9eFzNDigkUigldEYIGHAoRBftxbmUN2uA8ufiQTi+G/Ohf1Up7YGhQBZI+sjarl/VmEVDkxAJEMaGEBCJZtokFXURkERwiIujLJxgf0mdXWGTvL33aL6+mT1iSnKF/CxKICEbsgkEnESqhBAvEXjTimEX1DKlyySDe1JO6Da7r/AP5w81Ba1pJUx4BQzfzRCEO4kFNDSIkouEXdPaAkrtwvP/6+wHl/RPaQcQuGr7CIbGdv/LkdXrRjlfNHv3tqlvEuiOgWSLSdTCOrJhYxHlASQlGdLCAROJgs7KhFSWBIrbjpz2uNLbe62+JHYns0eODTZa44zFLLLzyrm5ih0jRkIBhLPh4XwZhMwwgeT8piAbL96+e+rjS2FokSJtYe95KWFZdm5HSwEJApMiP4pEGyQ95JCziMICycAjVoaAneDmX+iVRLGb+QFMkzSGTbG8Z4ltJiU57HQUR6V1gpEvexSd0OsQtsmUFkq2QVKg/+TRf6uy9OQ39+GoVx8weuUmdSP/SMHnUvHotkbyWaxcwqOiTnbMwEFExs+hB7z8je4ROL8Vj7YS177V9huS6WhbpSBQh4mH+KIfELgWOrJPCgUMcx4DkN9bzSPNB57vpT30mPDLMN2zzcbW4G7bHLCtaDoGJYwEGWGBBrbBgiQZyIiagJBEx+eIZHHbXk6SANIPM/TE1h9JDTGaPWbqls0Ig4YRV86obRJUwRGQbXXpu6bEwycWaPdBsYu0xSCGpqSUiwkEFDt+q3CFsUXRo5yWPBA5fJY91eBST53S4WruTvN3zfE4jTfJXdT2/quuUxQqDiAOf0u5dmPBR9qDFAn+waCrC/NncPeYXd8Gw9efaz2ND89/IIVbiEVUGkUMi0csjxWSXBQW2JMKSScQDGM80zHu4T74dFANVu4BM5EjkkR4hGwdBUyU5BAjWgkIBxWksJIs8+2klfXZ+GET6tBIGOupOmmfI0iAEejEpLIIRFnEeRzMk1POfVtLp5aHxWEdiLNoVZK9DaE2s0gijeJQq0HeHjiE3amf+yHqfNSKhL/8L0+p2lMxDo3A4ivs5hGoOSckfO7MISAoHFMCxCxi+df0nwmM/krp2q/Zrs7BIcwjd5lVziCxCxZb+WDfIl/+R8BijZDdzyZ3T47ro0EvpFtIzndod0mYRduW5SfNq1MUzvZj/pYtiN4itOyNRiPQMsSWH5B1E2hkW8XAdmTfCH9ZLHP9JdwjJbsUad6WxmWSIfncik1CSOaQsEjyiUfAoHDu5479ztJq65KBwsOZZvd8g7a4eoV486CUNCu2Mym5XOFgFo+p/Maxu6+zyINyRG/23Nf8Sd6MISWlmmUdkj1LxKCSX/+1htUCCS+AgG3gc3AFCzTnjQYuGQr1fQWARDYzGoqos8l+7d+zT6dXh8OTOEpV7O6So0AulN2qVxCV4XFz9v3DEKfhio3F1/Rdp6Pk9OHaHhGQQ+aMsQjUWiwi5/O8n+SxMDvMV/e7m0GNv/wa3G0Q4VIuJJYNcXJ79T3GYTs8v6rs/6ztJFtnjkM6DXnfI7n9wzP0LaXL39Limi+N4DTGT6OvCQSs85JD/Y3Ksp8nlQc0f9rtx+dMfZXUWQtL0v02OCZPdX+egWA+ERYRqj4vi4VsUWjL5fwfH9NR1x2y/pqdvi/Mjb7GQXmjMmJxdAuUuGYKG13e9LdJLGHRSERJy44XGnoy/3AnKPi4R6rRShETfQKNZxHGMg+r89IXGXwqUi4O9GeLdv8FNixSSMULoF2v8vel1dXG4mVvDN6pNrKSROGLTHSRZvMD4e1BOscpmzsSleRV7jKxmjxLGeBlT94NydnV5cTi/qYsINJJH/0UWssjAGC8wHsgrwiIW4iEkXoNDQPHiC+mh7cL82SjfqbWXLNo8YaZ4IfHIgsvp2fn51dXl5cXFxc7ywfMjxhIMLi+vrs7Pz06f5Xj6A3DX+cnaEE5XAAAAAElFTkSuQmCC"/>
```

Comment:
Hmm, this seems weird. I don't think there should be a base 64 image in an svg.

The status-logo SVG for exmaple is way simpler: 
https://github.com/status-im/status-desktop/blob/e279e639a488ce4ca0b8a9cb28f6063d888a92ee/ui/imports/assets/icons/status-logo.svg

@xAlisher can you look again how the SVG was made? It seems like the background is an image

Reply:
The header clearly says image/png do it's just an embedded PNG in an SVG 😑